DashWallet/Sources/UI/Coinbase/Order Preview/OrderPreviewViewController.swift

```diff
@@ -70,7 +70,17 @@ class OrderPreviewViewController: BaseViewController, NetworkReachabilityHandlin
 
     @objc
     func feeInfoAction() {
-        let nvc = BaseNavigationController(rootViewController: CoinbaseFeeInfoController())
+        let vc = BasicInfoController()
+        vc.mainAction = { UIApplication.shared.open(kCoinbaseFeeInfoURL) }
+        vc.icon = "coinbase.fee.info"
+        vc.headerText = "Fees in crypto purchases"
```

Comment:
@Syn-McJ Seems like I forgot to wrap these strings `NSLocalizedString`, could you please add it here and for `descriptionText` too. I will fix it on develop branch within `CoinbaseFeeInfoController` as well.

Reply:
Done